src/lib/components/Carousel/index.js

```diff
@@ -60,90 +49,98 @@ class Carousel extends BaseElement {
       ...this._carouselTrack.children,
     ]);
 
-    this._items.forEach((v, i) => v.setAttribute('data-index', '' + i));
-
     this._previousButton = this.querySelector('button[data-direction="prev"]');
-    this._previousButton.addEventListener('click', this.previous);
+    this._previousButton.addEventListener('click', this._previous);
+
     this._nextButton = this.querySelector('button[data-direction="next"]');
-    this._nextButton.addEventListener('click', this.next);
+    this._nextButton.addEventListener('click', this._next);
 
     this._items.forEach((e, i) =>
-      e.addEventListener('focusin', () => this.moveSlide(i - this._index)),
+      e.addEventListener('focusin', () => this._moveSlide(i - this._index)),
     );
+    this._carouselTrack.addEventListener('keyup', this._onKeyup);
 
-    this._carouselTrack.addEventListener('scroll', this._onScroll);
-
-    this.moveSlide(0, false);
+    this._moveSlide(0);
```

Comment:
Why this? It gives the focus on first card after render, even if no user action indicated they indicated to focus that card. 

It gives an additional side effect, that if they load the page, dont even scroll to the carousel, and hit accidentally enter, they will navigate to a random page.

Reply:
This is a total remnant of the way it used to work, it definitely shouldn't be here now.